src/main/java/FlowerStore/Main.java

```diff
@@ -7,24 +7,28 @@
 
 public class Main {
     public static void main(String[] args) {
-        ItemManager list = new ItemManager();
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 55, "special", "blue"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "pink"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "pink"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.TULIP, 35, "fine", "red"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.DAISY, 44, "average", "white"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.IRIS, 62, "great", "purple"));
-        list.addItem(new BouquetFlower(BouquetFlowerType.PEONY, 70, "perfect", "pink"));
-        System.out.println(list.toString());
-        /*list.sortByField("itemName", SortOrder.ASC);
-        System.out.println(list.toString());
-        list.sortByField("price", SortOrder.DESC);
-        System.out.println(list.toString());*/
         ItemManager manager = new ItemManager();
-        manager.compileBouquet(new BouquetConfiguration("red", new PriceRange(0, 50), Arrays.asList("ROSE", "TULIP", "DAISY"), 3, "blue paper"), list.getItems());
-        System.out.println(manager.toString());
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 55, "special", "blue"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "red"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "pink"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.ROSE, 50, "great", "pink"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.TULIP, 35, "fine", "red"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.DAISY, 44, "average", "white"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.IRIS, 62, "great", "purple"));
+        manager.addItem(new BouquetFlower(BouquetFlowerType.PEONY, 70, "perfect", "pink"));
+
+        BouquetConfiguration bouquetConfiguration = new BouquetConfiguration("red",
+                new PriceRange(0, 50),
+                Arrays.asList("ROSE", "TULIP", "DAISY"),
+                3,
+                "blue paper"
+        );
```

Comment:
In reality, having constructors with more than four arguments is not the best practice. 
I recommend you use the pattern called Builder. 
Otherwise, you can also check 
```@Accessors(chain = true)``` from Lombok and configure it with setters.